### PR TITLE
Update readme to include buildbot information

### PR DESCRIPTION
**Pull request by fhinkel**
I updated our readme, it looks like this: https://github.com/fhinkel/node/tree/vee-eight-lkgr

Can you take a look please? Thanks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,409 +1,12 @@
 # Node.js
 
-[![Gitter](https://badges.gitter.im/Join Chat.svg)](https://gitter.im/nodejs/node?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/29/badge)](https://bestpractices.coreinfrastructure.org/projects/29)
+This is the official [V8](https://developers.google.com/v8/) fork of Node.js with a recent (usually Canary) V8 version.
```
Comment (natorion) [inline README.md]: please link to www.v8project.org instead.

Comment (fhinkel) [inline README.md]: Done

File: README.md [hunk 2]
```diff
@@ -1,409 +1,12 @@
 # Node.js
 
-[![Gitter](https://badges.gitter.im/Join Chat.svg)](https://gitter.im/nodejs/node?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/29/badge)](https://bestpractices.coreinfrastructure.org/projects/29)
+This is the official [V8](https://github.com/v8/v8/wiki) fork of Node.js with a recent (usually Canary) V8 version. 
 
-Node.js is a JavaScript runtime built on Chrome's V8 JavaScript engine. Node.js
-uses an event-driven, non-blocking I/O model that makes it lightweight and
-efficient. The Node.js package ecosystem, [npm][], is the largest ecosystem of
-open source libraries in the world.
+Download the executable for Ubuntu from this [build bot](https://build.chromium.org/p/client.v8.fyi/builders/V8%20-%20node.js%20integration). Select a build, then use *Archive link download*.
```
Comment (hashseed) [inline README.md]: I'd write "latest builds" instead of "executable".

Comment (fhinkel) [inline README.md]: Done

**Comment by hashseed** [comment]
lgtm.

**Pull request merged**